Controllers/DoctorsController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Profiles.Data;
+using Profiles.Models;
+
+namespace ProfilesMicroService.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class DoctorsController : Controller
+    {
+        // Приватное поле для хранения контекста базы данных.
+        private readonly ApplicationDbContext _context;
```

Comment:
let's make a service(inherited from the interface) that hides the implementation of accessing the database